src/components/AvatarGroup/AvatarGroupCounterTooltipContainer.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Dialog from "../Dialog/Dialog";
+import AvatarGroupCounterTooltipContent from "./AvatarGroupCounterTooltipContent";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+
+const AvatarGroupCounterTooltipContainer = ({ children, avatars, type, className, counterTooltipCustomProps }) => {
+  const counterTooltipContentExists = !!avatars?.length;
+  const counterTooltipContentComponent = (
+    <AvatarGroupCounterTooltipContent avatars={avatars} type={type} className={className} />
+  );
+
+  if (!counterTooltipContentExists && !counterTooltipCustomProps) {
+    return children;
+  }
+
+  if (counterTooltipCustomProps) {
+    return (
+      <Tooltip
+        showTrigger={[Dialog.hideShowTriggers.FOCUS, Dialog.hideShowTriggers.MOUSE_ENTER]}
+        hideTrigger={[Dialog.hideShowTriggers.TAB_KEY, Dialog.hideShowTriggers.MOUSE_LEAVE]}
+        showOnDialogEnter
+        hideDelay={200}
+        {...counterTooltipCustomProps}
+      >
+        {children}
+      </Tooltip>
+    );
+  }
+
+  return (
+    <Tooltip
+      content={counterTooltipContentComponent}
+      showOnDialogEnter
+      hideDelay={200}
+      showTrigger={[Dialog.hideShowTriggers.FOCUS, Dialog.hideShowTriggers.MOUSE_ENTER]}
+      hideTrigger={[Dialog.hideShowTriggers.TAB_KEY, Dialog.hideShowTriggers.MOUSE_LEAVE]}
+    >
+      {children}
+    </Tooltip>
+  );
+};
+
+AvatarGroupCounterTooltipContainer.propTypes = {
+  className: PropTypes.string,
+  type: PropTypes.oneOf([Avatar.types.TEXT, Avatar.types.ICON, Avatar.types.IMG]),
+  /**
+   * Counter element
```

Comment:
```suggestion
   * Avatar Counter element
```

because it is not a regular counter element right?